hackathon_site/event/urls.py

```diff
@@ -3,4 +3,4 @@
 
 app_name = "event"
 
-urlpatterns = [path("", IndexView.as_view())]
+urlpatterns = [path("", IndexView.as_view(), name="index")]
```

Comment:
shouldn't it be 

`urlpatterns = [path("/", IndexView.as_view(), name="index")]`

Reply:
Django treats that as a leading slash, and will raise a warning. Even if you navigate to `/` in the browser, this route will capture it.